test/test_vmap.py

```diff
@@ -2438,10 +2434,6 @@ def test_view_as(self):
         B0, B1, B2 = 7, 11, 13
         op = torch.Tensor.view_as
 
-        # We should error out if the view would produce an incorrect result
```

Comment:
Why did you remove these tests?

Reply:
I'm not sure, but if this check is present I have locally:
```
        # We should error out if the view would produce an incorrect result                                                                                                                         with self.assertRaises(RuntimeError):                                                                                                                                               
>           vmap(op, in_dims=(1, 0))(torch.rand(2, B0, 5), torch.rand(B0, 10))                                                                                                              
E           AssertionError: RuntimeError not raised
```